docs/development/codeguide.rst

```diff
@@ -190,20 +190,12 @@ Coding Style/Conventions
             See :doc:`codeguide_emacs` for some configuration options for Emacs
             that helps in ensuring conformance to PEP8.
 
-* Astropy source code should contain a comment at the beginning of the file (or
```

Comment:
I could not find a single such shebang in the code base (excluding `extern/`), which makes sense since astropy isn't a collection of scripts. So I figure this precision is best left omitted.